package.json

```diff
@@ -17,6 +17,23 @@
   "directories": {
     "test": "test"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged",
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
+    }
+  },
+  "lint-staged": {
+    "*.js": [
+      "yarn lint --fix",
+      "yarn format"
+    ]
+  },
```

Comment:
Good job here, but the package is in typescript, so shouldn't lint fetch .ts files? And I'm sorry, I don't know this `yarn format` command, would it be necessary to add a .eslintignore and add the **build** and **node_modules** folders?

Reply:
Hey, great point. I can add support for `.ts` on a separate PR, that can be merged to the branch `feature/adonis5-alpha`.

About the `build` and `node_modules`, the command will ignore everything from the `.gitignore` with the `--ignore-path` flag. I've added it on the previous PR that configures Prettier on the project.